src/lib/Player.ts

```
@@ -3,229 +3,263 @@
+	 */
+	public async play(
+		track: string,
+		options?: Omit<UpdatePlayerData, "track"> & {
+			noReplace?: boolean;
+			userData?: Record<any, any>;
+		}
+	): Promise<UpdatePlayerResult> {
+		const noReplace = options?.noReplace ?? false;
+		const userData = options?.userData;
+		if (options) {
+			delete options.noReplace;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)